sdk/network/Microsoft.Azure.Management.Network/src/Microsoft.Azure.Management.Network.csproj

```diff
@@ -10,18 +10,16 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Network" (https://www.nuget.org/packages/Azure.ResourceManager.Network/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>21.0.0</Version>
```

Comment:
So you're no going to merge this to `main` right? because you're downgrading the version number

Reply:
No, we will use our own independent package before GA.